Dockerfile

```diff
@@ -45,4 +45,5 @@ FROM gcr.io/distroless/static:latest
 WORKDIR /
 COPY --from=builder /workspace/manager .
 USER nobody
-ENTRYPOINT ["/manager"]
+ENV NAMESPACE ""
+ENTRYPOINT ["/manager", "--namespace=${NAMESPACE}"]
```

Comment:
Please remove the parameter option here. 
Pass it via:
https://github.com/kubernetes-sigs/application/blob/master/config/manager/manager.yaml#L46

Reply:
done. and i think this file can be configured as needed.